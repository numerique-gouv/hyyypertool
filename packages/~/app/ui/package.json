{
  "name": "@~/app.ui",
  "version": "1.0.0",
  "private": true,
  "type": "module",
  "exports": {
    "./button": {
      "default": "./src/button/index.ts"
    },
    "./button/scripts": {
      "default": "./src/button/scripts/index.ts"
    },
    "./callout": {
      "default": "./src/callout/index.ts"
    },
    "./form": {
      "default": "./src/form/index.ts"
    },
    "./hx_table": {
      "default": "./src/hx_table/index.ts"
    },
    "./loader": {
      "default": "./src/loader/index.ts"
    },
    "./menu": {
      "default": "./src/menu/index.ts"
    },
    "./notice": {
      "default": "./src/notice/index.ts"
    },
    "./plurial": {
      "default": "./src/plurial/index.ts"
    },
    "./quote": {
      "default": "./src/quote/index.ts"
    },
    "./table": {
      "default": "./src/table/index.ts"
    },
    "./visually_hidden": {
      "default": "./src/visually_hidden/index.ts"
    },
    "./*": {
      "default": "./src/*.tsx"
    }
  },
  "scripts": {
    "test": "bun test"
  },
  "dependencies": {
    "@~/app.core": "workspace:*",
    "@~/app.urls": "workspace:*",
    "hono": "4.5.11",
    "hono-slotify": "workspace:*",
    "tailwind-variants": "0.2.1"
  },
  "devDependencies": {
    "@~/config.happydom": "workspace:*",
    "@~/config.tailwindcss": "workspace:*",
    "@~/config.typescript": "workspace:*"
  }
}
