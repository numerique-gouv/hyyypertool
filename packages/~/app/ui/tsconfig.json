{
  "compilerOptions": {
    "jsx": "react-jsx",
    "jsxImportSource": "hono/jsx",
    "lib": ["DOM"],
    "outDir": "./node_modules/.cache/tsc",
    "types": ["bun-types", "hono"]
  },
  "extends": "@~/config.typescript/tsx/tsconfig.json",
  "references": [
    {
      "path": "../core"
    },
    {
      "path": "../urls"
    },
    {
      "path": "../../../hono-slotify"
    },
    {
      "path": "../../config/happydom"
    }
  ]
}
