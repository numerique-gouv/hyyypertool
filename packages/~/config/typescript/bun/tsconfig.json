{
  "$schema": "https://json.schemastore.org/tsconfig",
  "display": "Bun",
  "extends": "../base/tsconfig.json",
  "compilerOptions": {
    "types": ["bun-types"]
  }
}
