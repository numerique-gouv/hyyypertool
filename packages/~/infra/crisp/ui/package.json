{
  "name": "@~/crisp.ui",
  "version": "1.0.0",
  "private": true,
  "type": "module",
  "exports": {
    ".": {
      "default": "./src/index.ts"
    },
    "./message": {
      "default": "./src/Message/index.tsx"
    },
    "./*": {
      "default": "./src/*.tsx"
    }
  },
  "dependencies": {
    "@~/crisp.lib": "workspace:*",
    "rehype-sanitize": "6.0.0",
    "rehype-stringify": "10.0.1",
    "remark-parse": "11.0.0",
    "remark-rehype": "11.1.1",
    "ts-pattern": "5.4.0",
    "unified": "11.0.5"
  },
  "devDependencies": {
    "@~/config.tailwindcss": "workspace:*",
    "@~/config.typescript": "workspace:*"
  }
}
