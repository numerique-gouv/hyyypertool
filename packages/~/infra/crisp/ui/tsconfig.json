{
  "compilerOptions": {
    "jsx": "react-jsx",
    "jsxImportSource": "hono/jsx",
    "outDir": "./node_modules/.cache/tsc"
  },
  "extends": "@~/config.typescript/tsx/tsconfig.json",
  "references": [
    {
      "path": "../lib"
    }
  ]
}
