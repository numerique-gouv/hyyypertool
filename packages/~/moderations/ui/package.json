{
  "name": "@~/moderations.ui",
  "version": "1.0.0",
  "private": true,
  "type": "module",
  "imports": {
    "#ui/*": {
      "types": "./src/*/index.ts",
      "default": "./src/*/index.ts"
    }
  },
  "exports": {
    "./*": {
      "types": "./src/*/index.ts",
      "default": "./src/*/index.ts"
    }
  },
  "dependencies": {
    "@~/app.ui": "workspace:*",
    "@~/app.urls": "workspace:*",
    "@~/organizations.lib": "workspace:*",
    "@~/users.lib": "workspace:*",
    "@~/moderations.lib": "workspace:*",
    "hono": "4.6.9",
    "tailwind-variants": "0.2.1",
    "ts-pattern": "5.4.0"
  },
  "devDependencies": {
    "@~/crisp.lib": "workspace:*",
    "@~/config.typescript": "workspace:*"
  }
}
