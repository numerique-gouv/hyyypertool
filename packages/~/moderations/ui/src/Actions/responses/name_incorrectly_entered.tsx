//

import { useContext } from "hono/jsx";
import { dedent } from "ts-dedent";
import { context } from "../context";

export const label =
  "Nom et/ou prénom mal renseignés - Modération non-bloquante";

export default function template() {
  const {
    moderation: {
      organization: { siret },
    },
  } = useContext(context);
  return dedent`
    Bonjour, 

    Votre nom et/ou prénom n'a pas été correctement renseigné.

    Les champs Nom et Prénom doivent correspondre à ceux d’une personne physique.

    Merci de bien vouloir corriger cela depuis la section « Vos informations personnelles » de votre compte ProConnect (anciennement : AgentConnect, MonComptePro).

    Pour se faire :

    - Accédez à votre compte ProConnect en cliquant sur le lien suivant : https://app.moncomptepro.beta.gouv.fr/users/personal-information,
    - Authentifiez-vous si cela vous est demandé par l'interface,
    - Corrigez votre prénom et votre nom,
    - Sélectionnez votre organisation (numéro SIRET : ${siret}) pour que notre équipe puisse valider à nouveau la création de votre compte.

    Sans cette correction, nous serons contraints de supprimer votre compte prochainement.

    Merci pour votre correction.
    Bien cordialement
    L’équipe ProConnect.
  `;
}
