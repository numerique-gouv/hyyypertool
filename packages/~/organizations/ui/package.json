{
  "name": "@~/organizations.ui",
  "version": "1.0.0",
  "private": true,
  "type": "module",
  "exports": {
    "./*": {
      "default": "./src/*.tsx"
    }
  },
  "dependencies": {
    "@~/app.ui": "workspace:*",
    "@~/app.urls": "workspace:*",
    "@~/organizations.lib": "workspace:*",
    "hono": "4.7.11",
    "tailwind-variants": "0.3.1",
    "ts-pattern": "5.7.0",
    "zod": "3.24.1"
  },
  "devDependencies": {
    "@~/config.typescript": "workspace:*"
  }
}
