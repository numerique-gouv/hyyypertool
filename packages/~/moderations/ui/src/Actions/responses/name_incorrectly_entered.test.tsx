//

import { render_md } from "@~/app.ui/testing";
import { expect, test } from "bun:test";
import name_incorrectly_entered from "./name_incorrectly_entered";

//

test("returns incorrect name entered response", async () => {
  expect(await render_md(<Response />)).toMatchSnapshot();
});

function Response() {
  return <>{name_incorrectly_entered()}</>;
}
