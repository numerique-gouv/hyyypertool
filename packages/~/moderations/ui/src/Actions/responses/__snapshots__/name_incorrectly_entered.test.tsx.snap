// Bun Snapshot v1, https://goo.gl/fbAQLP

exports[`returns incorrect name entered response 1`] = `
"Bonjour,

Votre nom et/ou prénom n&#39;a pas été correctement renseigné.

Les champs Nom et Prénom doivent correspondre à ceux d’une personne physique.

Merci de bien vouloir corriger cela depuis la section « Vos informations personnelles » de votre compte ProConnect (anciennement : AgentConnect, MonComptePro).

Sans cette correction, nous serons contraints de supprimer votre compte prochainement.

Merci pour votre correction.
Bien cordialement
L’équipe ProConnect.
"
`;
